recipe/meta.yaml

```diff
@@ -8,9 +8,11 @@ source:
   fn: sympy-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/s/sympy/sympy-{{ version }}.tar.gz
   md5: c410a9c2346878716d16ec873d72e72a
```

Comment:
Do you mind updating this to the safer `sha256`? You can get it from the new PyPI site.

Reply:
Sure :) Fixed.